### ISSUE TITLE
'opera_app' environment fails on M1 Mac

### ISSUE THREAD

**Issue by EJFielding** [OPERA-Cal-Val/OPERA_Applications#26]
I was not able to install the `opera_app` environment on my M1 Mac running macOS 13.4.1. It seems that the YAML file specifies `python=3.7` and that is not available on the M1 Mac. Does the environment only work with Python v3.7 that is now at the end of support?
```
fielding@MT-xxxxxx OPERA_Applications % conda env create -f environment.yml
Retrieving notices: ...working... done
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - python=3.7
  - nodejs=14.18.3
```

**Comment by EJFielding** [comment]
I am still not able to install the environment on my M1 Mac because of the `nodejs=14.18.3` not being available. The oldest version available on my M1 Mac is 15.8.0.
```
conda search nodejs
Loading channels: done
# Name                       Version           Build  Channel
nodejs                        15.8.0      h3f26200_1  conda-forge
nodejs                       15.10.0      h3f26200_0  conda-forge
...
```

**Comment by cmspeed** [comment]
Issue solved with nodejs update to version 18.16.1.
